temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -105,6 +105,29 @@ service CloudService {
         };
     }
 
+    // Failover a multi-region namespace
+    rpc FailoverNamespace (FailoverNamespaceRequest) returns (FailoverNamespaceResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/namespaces/{namespace}/failover",
+            body: "*"
+        };
+    }
+
+    // Get failover history on a namespace
+    rpc GetNamespaceFailoverHistory (GetNamespaceFailoverHistoryRequest) returns (GetNamespaceFailoverHistoryResponse) {
```

Comment:
Why do we need this? Doesn't the audit logs capture this?

Reply:
I think this is more from a debugging/dc stand point to get the failover history. But I don't have a strong opinion. @atihkin do you think exposing this is necessary or we can defer this.

Reply:
We're showing this info in the UI so I think it should show up here as well.

Reply:
These APIs are meant for automation use cases. Having parity with the UI is not really necessary. 
Though i am not really against having this specific API, but I would side on adding only if there is really a automation use case around it.

Reply:
@atihkin If the cloud api is for automation use cases, I don't think we want to support them when launching?